contracts/minter-suite/Minters/MinterSEAV0.sol

```diff
@@ -0,0 +1,1186 @@
+     * only result in a gas cost increase, since the core contract will still
+     * enforce a maxInvocation check during minting. A false positive is not
+     * possible because the V3 core contract only allows maximum invocations
+     * to be reduced, not increased. Based on this rationale, we intentionally
+     * do not do input validation in this method as to whether or not the input
+     * `_projectId` is an existing project ID.
+     *
+     */
+    function projectMaxHasBeenInvoked(
+        uint256 _projectId
+    ) external view returns (bool) {
+        return projectConfig[_projectId].maxHasBeenInvoked;
+    }
+
+    /**
+     * @notice projectId => project's maximum number of invocations.
+     * Optionally synced with core contract value, for gas optimization.
+     * Note that this returns a local cache of the core contract's
+     * state, and may be out of sync with the core contract. This is
+     * intentional, as it only enables gas optimization of mints after a
+     * project's maximum invocations has been reached.
+     * @dev A number greater than the core contract's project max invocations
+     * will only result in a gas cost increase, since the core contract will
+     * still enforce a maxInvocation check during minting. A number less than
+     * the core contract's project max invocations is only possible when the
+     * project's max invocations have not been synced on this minter, since the
+     * V3 core contract only allows maximum invocations to be reduced, not
+     * increased. When this happens, the minter will enable minting, allowing
+     * the core contract to enforce the max invocations check. Based on this
+     * rationale, we intentionally do not do input validation in this method as
+     * to whether or not the input `_projectId` is an existing project ID.
+     */
+    function projectMaxInvocations(
+        uint256 _projectId
+    ) external view returns (uint256) {
+        return uint256(projectConfig[_projectId].maxInvocations);
+    }
+
+    /**
+     * @notice projectId => project configuration details.
+     * Note that in the case of no auction being initialized for the project,
+     * the returned `auction` will be the default struct.
+     * @param _projectId The project ID
+     * @return maxInvocations The project's maximum number of invocations
+     * allowed on this minter
+     * @return timestampStart The project's start timestamp, after which new
+     * auctions may be created (one at a time)
+     * @return auctionDurationSeconds The project's default auction duration,
+     * before any extensions due to buffer time
+     * @return basePrice The project's minimum starting bid price
+     * @return nextTokenNumberIsPopulated Whether or not the project's next
+     * token number has been populated
+     * @return nextTokenNumber The project's next token number to be auctioned,
+     * dummy value of 0 if `nextTokenNumberIsPopulated` is false
```

Comment:
Nit: 

`0` may not be the best dummy value here, as could be a real value for a new Engine project starting w/ project 0...

That said, it is probably OK if this caveat is just noted as a quirk of this function in the non-initialized state via dev comment, as I'm not sure what a better dummy value would be...

Reply:
agree that `0` could certainly be a real token's ID. I think `0` is probably the most ideal dummy value here, since it is Solidity's default value (and most gas-efficient value to return).
Definitely a quirk though 😅

Reply:
Roger roger – I think worth just noting in dev comment that `0` could be a real value so shouldn't be relied on as way to check false-y/dummy value state (though that this _is_ the dummy value).

Reply:
good call, updated the comment!